README.md

```diff
@@ -15,7 +15,7 @@ See my blog post on "[Flame Graphs with Java Flight Recordings]" for more detail
 
 ## How to build
 
-Call `./gradlew assemble`
+Call `./gradlew clean assemble`
```

Comment:
This is to avoid the possibility the `*` in the script matching multiple JARs that might accumulate from non-clean builds after the version number is bumped.